kvmagent/kvmagent/plugins/hygon_device_plugin.py

```
@@ -558,8 +567,106 @@ def generate_hygon_mdev_devices(self, req):
+            new_content = content[:match.start()] + new_cgroup_device_acl + content[match.end():]
+
+            # Write back to file
+            with open(qemu_conf_path, 'w') as f:
+                f.write(new_content)
+
+            logger.info("Successfully appended %s to cgroup_device_acl in %s" %
+                       (hygon_psp_config_device, qemu_conf_path))
+
+        except IOError as e:
+            logger.warning("Failed to read/write %s: %s" % (qemu_conf_path, str(e)))
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)